src/convergence judgment.ts

```diff
@@ -16,11 +16,9 @@ limitations under the License.
 function convergence_judgment() {
   function currentvalue(n: number) {
     sheet.getCurrentCell().offset(n, 0).activate().getvalue();
```

Comment:
こうですかね？ `getvalue` は存在しないってエラーになってたので
```suggestion
    sheet.getCurrentCell().offset(n, 0).activate().getValue();
```

Reply:
Typo だと思います．修正しておきます

Reply:
:ok: